pennylane/gradients/parameter_shift.py

```diff
@@ -927,7 +940,7 @@ def processing_fn(results):
 
 
 @gradient_transform
-def param_shift(
+def _param_shift_new(

```

Comment:
Worthwhile to update the examples in this docstring, or at least mention that this function replaces `param_shift` in the new return types system.

Reply:
Added TODO and a comment.